packages/slice-machine/package.json

```diff
@@ -20,125 +16,110 @@
     "dev": "next dev",
     "dev-cypress": "concurrently \"next dev\" \"npm run dev --prefix ../../e2e-projects/cypress-next-app\"",
     "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
-    "prepublishOnly": "npm run build",
+    "prepack": "$npm_execpath run build",
     "storybook": "storybook dev --no-open --port 6006",
     "storybook-build": "storybook build",
-    "test": "npm run lint && npm run types && npm run unit && npm run build",
+    "test": "yarn lint && yarn types && yarn unit && yarn build",
     "types": "tsc --noEmit",
     "unit": "vitest run --coverage",
     "unit:watch": "vitest watch",
     "lint:precommit": "eslint",
     "depcheck": "depcheck --config=.depcheckrc",
-    "audit": "better-npm-audit audit -l high -p"
+    "audit": "yarn npm audit --environment production --severity high"
   },
-  "dependencies": {
-    "@extractus/oembed-extractor": "^3.1.8",
-    "@prismicio/editor-fields": "0.4.2",
-    "@prismicio/editor-support": "0.4.1",
-    "@prismicio/editor-ui": "0.4.2",
-    "@prismicio/mocks": "2.0.0-alpha.2",
-    "@prismicio/types-internal": "2.0.0-beta.1",
-    "@slicemachine/manager": "0.4.1-dev-next-release.5",
-    "@vanilla-extract/css": "1.11.0",
-    "clsx": "1.2.1",
-    "fast-deep-equal": "^3.1.3",
-    "fp-ts": "^2.13.1",
-    "html-react-parser": "^3.0.12",
-    "io-ts": "^2.2.20",
-    "io-ts-types": "^0.5.19",
-    "lodash": "^4.17.21",
-    "monocle-ts": "^2.3.13",
-    "newtype-ts": "^0.3.5",
-    "puppeteer": "^19.11.0",
-    "react": "^18.2.0",
-    "semver": "^7.3.8",
-    "start-slicemachine": "0.9.1-dev-next-release.5",
-    "yup": "^0.32.11"
+  "bin": {
+    "start-slicemachine": "./bin/start-slicemachine.cjs"
   },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
looks like a lot of the bundled deps will need to be re downloaded now?

Reply:
Okay, last time i checked we put bundled deps in the dev deps secton (things like react, and react-dom, next does this by default) :)